inst/rmarkdown/templates/oup/resources/template.tex

```diff
@@ -1,8 +1,31 @@
+% Colors for colorlinks
 $if(colorlinks)$
 \PassOptionsToPackage{dvipsnames,svgnames*,x11names*}{xcolor}
 $endif$
 
-\documentclass[$if(fontsize)$$fontsize$,$else$12pt,$endif$$if(spacing)$$spacing$,$else$halfline,$endif$$if(papersize)$$papersize$,$else$a4paper,$endif$$for(classoptions)$$classoptions$$sep$,$endfor$]{ouparticle}
+% Set document class options
+\documentclass[webpdf$if(papersize)$,$papersize$$else$,large$endif$$if(document_style)$,$document_style$$else$,contemporary$endif$$if(unnumsec)$,unnumsec$endif$$if(namedate)$,namedate$endif$$if(classoptions)$,$for(classoptions)$$classoptions$$sep$,$endfor$$endif$]{oup-authoring-template}
+
+% one column
+$if(onecolumn)$\onecolumn$endif$
+
+%\usepackage{showframe}
+
+% line numbers
+$if(numberlines)$\usepackage{lineno}\linenumbers$endif$
+
+% Theorem stuff from template
+\theoremstyle{thmstyleone}%
+\newtheorem{theorem}{Theorem}%  meant for continuous numbers
+%%\newtheorem{theorem}{Theorem}[section]% meant for sectionwise numbers
+%% optional argument [theorem] produces theorem numbering sequence instead of independent numbers for Proposition
+\newtheorem{proposition}[theorem]{Proposition}%
+%%\newtheorem{proposition}{Proposition}% to get separate numbers for theorem and proposition etc.
+\theoremstyle{thmstyletwo}%
+\newtheorem{example}{Example}%
+\newtheorem{remark}{Remark}%
+\theoremstyle{thmstylethree}%
+\newtheorem{definition}{Definition}
 
 % Packages I think are necessary for basic Rmarkdown functionality
```

Comment:
This list is largely because I built my original OUP template off of another existing rticles template (can't remember which) without getting to know the default pandoc latex template. Now that I know that, I see that there are potentially improvements to be made, but am not too enthusiastic about making them as everything is working. If I see easy cleanup opportunities, I will do it, otherwise, I will leave it.